charts/internal/shoot-cert-management-seed/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if gt (int .Values.replicaCount) 1 }}
```

Comment:
```suggestion
```

@timuthy Is this line too much?

Reply:
Yes, thanks for noticing. Copy-paste issue 🙃